packages/editor/src/components/post-text-editor/style.scss

```diff
@@ -1,11 +1,13 @@
-textarea.editor-post-text-editor {
+.editor-post-text-editor textarea {
```

Comment:
This is necessary because `TextareaControl` has a couple of wrappers around the `textarea` element.